compute/admin_guide/vulnerability_management/registry_scanning.adoc

```diff
@@ -104,6 +104,11 @@ Most vendors comply with the Docker Registry version 2 API.
 
 *Harbor*: specify the FQDN of your Harbor registry (\https://).
 
+*Nexus Registry:*
+<http\|https://<nexus_hostname>:<HTTP/HTTPS connector port for the specific Nexus repo>
```

Comment:
should this say "
specify the FQDN and connector port for your Sonatype Nexus registry.
And remove <http\|https://<nexus_hostname>:<HTTP/HTTPS connector port for the specific Nexus repo> because you have an example?